.github/workflows/publish.yml

```diff
@@ -0,0 +1,114 @@
+name: Build & Publish Electron Releases
+# This workflow builds and publishes releases of MLVET (available under 'Releases')
+# for Windows, Linux and MacOS.
+
+on:
+  # This workflow will run when the PR is merged into UAT from another branch (develop)
+  push:
+    branches: ['uat']
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # Mac (including M1)
+  build_on_mac:
+    runs-on: macos-latest
+    steps:
+      # Checkout this branch
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}
+
+      # Installing node.js
+      - uses: actions/setup-node@master
+        with:
+          node-version: 16
+
+      # Installing dependencies with higher timeout in case of slow network
+      - name: Install dependencies
+        run: yarn install:slow
+
+      # Build and publish electron
+      - name: Build Electron
+        env:
+          ELECTRON: true
+          PUBLISH_FOR_PULL_REQUEST: false
+          ELECTRON_CACHE: $HOME/.cache/electron
+          ELECTRON_BUILDER_CACHE: $HOME/.cache/electron-builder
+          USE_HARD_LINKS: false
+          YARN_GPG: no
+          GITHUB_TOKEN: ${{ secrets.github_token }}
+        run: yarn run publish:mac
+
+      # List the build files
+      - name: List all files in the release directory
+        run: ls ./release/build
+
+  # Windows
+  build_on_win:
+    runs-on: windows-latest
+    steps:
+      # Checkout this branch
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}
+
+      # Installing node.js
+      - uses: actions/setup-node@master
+        with:
+          node-version: 16
+
+      # Installing dependencies with higher timeout in case of slow network
+      - name: Install dependencies
+        run: yarn install:slow
+
+      # Build and publish electron
+      - name: Build Electron
+        env:
+          ELECTRON: true
+          PUBLISH_FOR_PULL_REQUEST: false
+          ELECTRON_CACHE: $HOME/.cache/electron
+          ELECTRON_BUILDER_CACHE: $HOME/.cache/electron-builder
+          USE_HARD_LINKS: false
+          YARN_GPG: no
+          GITHUB_TOKEN: ${{ secrets.github_token }}
+        run: yarn run publish:windows
+
+      # List the build files
+      - name: List all files in the release directory
+        run: ls ./release/build
+
+  # Linux
+  build_on_linux:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout this branch
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}
+
+      # Installing node.js
+      - uses: actions/setup-node@master
+        with:
+          node-version: 16
+
+      # Installing dependencies with higher timeout in case of slow network
```

Comment:
Why would the network be slow? If it's in a github actions runner it will have a 10 gigabit connection right